private/data/i18n/en.json

```diff
@@ -203,6 +204,12 @@
           "canceled": "Canceled",
           "refunded": "Refunded"
         },
+        "returnItemsTitle": "Return selected items",
+        "returnItemsApproveAlert": "You are about to return {{returnItemsQuantity}} items but have yet to approve or capture {{totalAmount}} payment on your order. Would you like to approve now?",
+        "returnItemsCaptureAlert": "Capture payment for total amount {{totalAmount}}?",
+        "returnItemsAlert": "Return {{returnItemsQuantity}} item(s), total return amount {{returnItemsTotal}}?",
+        "returnItemsSuccess": "Items have successfully been returned and refund initiated",
+        "returnItemsWait": "Payment capture taking time try again in a few minutes",
```

Comment:
This is grammatically incorrect. Perhaps "Payment capture is taking too long. Try again in a few minutes"